sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0 (2022-12-21)
+Add `azure.confidentialledger.receiptverification` module for Azure Confidential Ledger write transaction receipt verification.
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Add `verify_receipt` and `verify_receipt_from_dict` function to verify write transaction receipts from a `Receipt` object or a receipt JSON object.
+- Add sample code to get and verify a write receipt from a running Confidential Ledger instance.
+- Update README with examples and documentation for receipt verification.
 
 ### Other Changes
+- Add dependency on Python `cryptography` library (`>= 2.1.4`)
+- Add tests for receipt verification models and receipt verification public methods.
+- Update code from REST API specs
+- Update `swagger/README.md`
```

Comment:
This doesn't need to be in the SDK release notes.

Reply:
Got it, will remove

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-python/pull/27265/commits/28f7642c76da39c07e66c9cfae35a7dff46807f0